config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml

```diff
@@ -33,8 +34,12 @@ periodics:
       command:
       - runner.sh
       args:
-      - make
-      - test
+      - bash
+      - -c
+      - |
+        apt-get update &&
+        apt-get install -y libsystemd-dev &&
```

Comment:
Can we add a command in the script for this? I don't think we want to have this logic in 2 different places.

Reply:
Sure. I can do that.

Reply:
Updated.